.github/workflows/update_credential_provider_script.yml

```diff
@@ -0,0 +1,37 @@
+name: Update credential provider script
```

Comment:
Didn't add a on command as I first want to run this script manually and in future modify python script to compare mercurial scm versions of the file we are fetching

Reply:
@isabelrios let me know if there are any changes required here